src/js/i18n/locales/en.json

```diff
@@ -25,7 +25,7 @@
   "Menu.ThreeOneOneMobileListItem.callText": "Call 311",
   "Menu.ThreeOneOneMobileListItem.onlineText": "Submit an Online Request",
   "Menu.PrivacyPolicyListItem.text": "Read About Privacy",
-  "Menu.MobileFooter.text": "Alpha.austin.gov is a work in progress. For the full City of Austin website, visit",
+  "Menu.MobileFooter.text": "Alpha.austin.gov is a work in progress. For the full City of Austin website, visit {citySiteLink}.",
```

Comment:
Ok, so it looks like Menu.MobileFooter.text should only include citySiteLink and not both city and project. Is showing different content for arabic vs the other languages just a stop gap for arabic testing tomorrow?